src/index.test.js

```diff
@@ -1,5 +1,6 @@
 import _ from 'lodash';
-import IOpipe from '../dist/iopipe.js';
+//import IOpipe from '../dist/iopipe.js';
```

Comment:
@pselle this is a big change, no?

Reply:
Good catch! Unintentional from local testing